File: front/src/mock/handlers/performance.ts
import { http, HttpResponse, delay } from 'msw'
import { performanceStats, performanceSamples } from '../data/performance'

export const performanceHandlers = [
  http.get('/api/performance/stats', async () => {
    await delay(300)
    return HttpResponse.json(performanceStats)
  }),

  http.get('/api/performance/samples', async ({ request }) => {
    await delay(300)
    const url = new URL(request.url)
    const bspType = url.searchParams.get('bsp_type') || ''
    let list = performanceSamples
    if (bspType) list = list.filter((s) => s.bsp_type === bspType)
    return HttpResponse.json(list)
  }),
]

File: WebAPI/routers/performance.py
# -*- coding: utf-8 -*-
"""
Performance 路由 — 买卖点绩效统计 API（stub）。
"""

from fastapi import APIRouter, Query

router = APIRouter(prefix="/api/performance", tags=["performance"])


@router.get("/stats")
async def performance_stats():
    """买卖点绩效统计。"""
    return []


@router.get("/samples")
async def performance_samples(bsp_type: str = Query("")):
    """绩效样本明细。"""
    return []